src/components/modals/CreateIncidentModal.tsx

```diff
@@ -16,7 +16,7 @@ const CreateIncidentModal: Component<Props> = ({
         getShouldDisplay={getShouldDisplay}
         setShouldDisplay={setShouldDisplay}
       >
-        <CreateIncidentForm />
+        <CreateIncidentForm setShouldDisplay={setShouldDisplay} />
```

Comment:
I feel like this param should have a more meaningful name. Something like `onCreateIncident`. What we really want is the ability for `CreateIncidentModal` to react to the creating on an incident, not have `CreateIncidentForm` be aware of the modal (that makes them tightly coupled).